src/crimson/net/Connection.h

```diff
@@ -126,6 +126,7 @@ class Connection : public seastar::enable_shared_from_this<Connection> {
 #endif
 
   /// send a message over a connection that has completed its handshake
+  virtual seastar::future<> send(MessageURef msg) = 0;
   virtual seastar::future<> send(MessageRef msg) = 0;
```

Comment:
Will need to drop this interface.

Reply:
This will happen as part of refactoring the internals of Messenger. This overload, as well as other ones, are necessary for this PR.